scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/benchmark/JavaBenchmark.java

```diff
@@ -106,7 +106,7 @@ public static void main(String[] args) {
                 model.runBatchInference();
                 result[i] = System.nanoTime() - currTime;
             }
-            printStatistics(result, modelName +"Batch");
+            printStatistics(result, modelName +"batch_inference");
```

Comment:
You can remove the modelName from the second parameter as well here. 
The Model Name will be aptly captured when reporting the results of the benchmark run as we'd know which script we are running it with :)